lib/pengine/bundle.c

```diff
@@ -889,9 +889,7 @@ mount_add(pe__bundle_variant_data_t *bundle_data, const char *source,
 
     mount->source = strdup(source);
     mount->target = strdup(target);
-    if (options) {
-        mount->options = strdup(options);
-    }
+    pcmk__copy_or_null(&mount->options, options);
```

Comment:
nothing newly introduced but we aren't checking if calloc above went OK.

Reply:
I think there's an option you can turn on to make coverity always report it, but I'm not sure why it sometimes does even without the option. I'll add the check here. At some point I want to go through and check all of them but it never seems worth the time.